guides/code-for-humans.md

```diff
@@ -0,0 +1,44 @@
+---
+explains: What we should think of when writing code so the most important computer we work with—the human brain—can parse it effectively
+---
+
+# How to code for humans
+
+**Good code is written for humans and executable by machines**. This means that the human brain understanding your code is just as important – if not more so – as a machine understanding it. By understanding what the human brain is good at, and what it is not so good at, we can write code that functions better and is more easily maintainable.
+
+With your code you are decribing functionality so precise it can be executed by machines. However those machines are not the ones tasked with fixing the bugs when they arise. The compiler doesn’t care if you call a variable `e` or `fatalErrorCode`. However, another human (or you in the future) might be tasked with fixing a breaking bug, at which point `e` might make it very difficult to figure out what is going on.
+
+Code is read more often than it's written and most developer time is spent chasing bugs. There is much to be gained from writing code that is easy to understand, both if you are familiar with the code or new to the team.
+
+Here are some basic priciples that can help you write better code.
+
+## Human coding guidelines
+
+### Single Responsibility Principle
+
+* Functions should do 1 thing, and do it well
+* Functions should be around a maximum of 10-15 lines
+* Functions should never exceed 25 lines
+
+Tip: Describe every function in a _docstring_, if you use “and” in your description you know it does two things and needs to be split up.
+
```

Comment:
Or even a little bit more general: Describe functions, classes and modules according to standards like docstrings, jsdoc or whatever standard is applicable to the project

Reply:
I've taken the specific _docstring_ out and replaced in with 'inline documentation'